packages/access/graphql/resolvers/User.js

```diff
@@ -0,0 +1,45 @@
+const debug = require('debug')('access:resolvers:User')
+
+const { Roles, transformUser } = require('@orbiting/backend-modules-auth')
+
+const grantsLib = require('../../lib/grants')
+const campaignsLib = require('../../lib/campaigns')
+
+module.exports = {
+  accessGrants: async (user, args, { user: me, pgdb }) => {
+    if (!Roles.userIsMeOrInRoles(user, me, ['admin', 'supporter'])) {
+      return null
+    }
+
+    const grants = await grantsLib.findByRecipient(user, pgdb)
+
+    const userIds = grants.map(grant => grant.granteeUserId)
+      .concat(grants.map(grant => grant.recipientUserId))
+
+    const users =
+      userIds.length > 0
+        ? await pgdb.public.users.find({ id: userIds })
+        : []
+
+    debug('accessGrants', { user: user.id, grants: grants.length, userIds })
+
+    return grants.map(grant => ({
+      ...grant,
+      users: users.map(transformUser)
+    }))
+  },
+  accessCampaigns: async (user, args, { user: me, pgdb }) => {
+    if (!Roles.userIsMeOrInRoles(user, me, ['admin', 'supporter'])) {
+      return null
+    }
+
+    const campaigns = await campaignsLib.findForGrantee(user, pgdb)
+
+    debug('accessCampaigns', { user: user.id, campaigns: campaigns.length })
+
+    return campaigns.map(campaign => ({
+      ...campaign,
+      user: transformUser(user)
```

Comment:
This is most likely double transforming and could lead to issues. The user here should already be transformed. To debug: `const isTransformed = !!user._raw` (don't branch in prd code just to test it locally)

Reply:
Where is this used? Is it even needed?